### DISCUSSION TITLE
Does Kubeflow Pipelines have a Java SDK or JAR package?

### DISCUSSION THREAD

**Discussion by YanSongCode** [kubeflow/pipelines#10841] [General]
if you have the anwser, please communicat with me ,thank you very much

**Comment by rimolive** [comment]
We don't have a Java SDK, but with the proto files under https://github.com/kubeflow/pipelines/tree/master/backend/api/v2beta1 I think it's possible to create a Java client, still not an SDK but it depends on what you plan to achieve.
